src/main/asciidoc/api_specifications.adoc

```diff
@@ -20,18 +20,18 @@ It is based on http://swagger.io/[Swagger^], which has been standardized as http
 [.rule, caption="Rule {counter:rule-number}: "]
 .OpenAPI
 ====
-Specifications of the API SHOULD be provided using https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md[*OpenAPI 2.0*] (aka Swagger) or https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md[*OpenAPI 3.0*].
-OpenAPI uses the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#schemaObject[OpenAPI Schema Object] to describe the JSON representation of resources, which is a variant of https://json-schema.org/specification-links.html#draft-4[*JSON Schema*], with some significant incompatibilities.
+Specifications of the API SHOULD be provided using https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md[*OpenAPI 2.0*] (aka Swagger) or https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md[*OpenAPI 3.0*].
+OpenAPI uses the https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#schemaObject[OpenAPI Schema Object] to describe the JSON representation of resources, which is a variant of https://json-schema.org/specification-links.html#draft-5[*JSON Schema*], with some significant incompatibilities.
 ====
 
-The OpenAPI 2.0 standard specifies how to define https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#schemaObject[Schema Objects]
-and https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#data-types[Data Types] and the differences with the JSON Schema standard.
-
 [NOTE]
-OpenAPI 3.0 fixes some shortcomings in OpenAPI 2.0.
-However, this guide has not been updated yet for OpenAPI 3.0 but most is still applicable.
+OpenAPI 3.1 fixes some shortcomings in OpenAPI 3.0. 
```

Comment:
Only mention OpenAPI 3.1 that it exists, but should not yet be used. Remove other specifics about OpenAPI 3.1.

Reply:
change implemented in latest commit